sw_utils/ipfs.py

```diff
@@ -238,10 +240,20 @@ async def remove(self, ipfs_hash: str) -> None:
 
 
 class IpfsFetchClient:
-    def __init__(self, endpoints: list[str], timeout: int = 60):
+    def __init__(
+            self,
+            endpoints: list[str],
+            timeout: int = 60,
+            request_timeout: int = 60
+    ):
         self.endpoints = endpoints
         self.timeout = timeout
+        self.request_timeout = request_timeout
 
+    @backoff_in_method(
+        RuntimeError,
```

Comment:
may be create custom exception? `like IPFSException`

Reply:
Lets postpone ipfs errors handling until next PR.